cmd/orb.go

```diff
@@ -217,7 +217,10 @@ func releaseOrb(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	Logger.Infof("Orb published %s", response.Orb.Version)
+	response.Orb.Namespace = args[1]
```

Comment:
Would it make more sense for `api.OrbPublishByID` to return an `OrbVersion` instead of an `OrbPublishResponse`? Then these attribute assignments on `Orb` don't need to be duplicated in `orb.go`.